sdk/core/azure-core/vcpkg.json

```diff
@@ -35,7 +35,8 @@
           "default-features": false,
           "features": [
             "ssl"
-          ]
+          ],
+          "version>=": "7.44"
```

Comment:
This change resulted in 7.44 showing up here:
https://github.com/microsoft/vcpkg/pull/32971

And looks like the oldest version of curl listed in the version databse within he catalog is 7.48:
https://github.com/microsoft/vcpkg/blob/master/versions/c-/curl.json#L425

So, to make that work, we got bumped up by vcpkg, here:
https://github.com/microsoft/vcpkg/pull/33109